openshift/kustomize/overlays/service-ca-operator/kustomization.yaml

```diff
@@ -0,0 +1,58 @@
+resources:
+- ../../../../manifests/base/apis
+- ../../../../manifests/base/core
+- ../../../../manifests/base/provisioners
+- ./resources/v1_configmap_rukapk-webhook-tls.yaml
+- ./resources/v1_configmap_rukpak-core-tls.yaml
+
+namespace: openshift-rukpak
+
+commonAnnotations: 
+ release.openshift.io/feature-set: "TechPreviewNoUpgrade"
```

Comment:
This isn't needed as part of the move to having cluster-olm-operator handle the lifecycle of rukpak (only the cluster-olm-operator manifests that the CVO processes need this annotation)

Reply:
Good to know, I'll remove it then. TY!